package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^5.3.4",
```

Comment:
did you handpick this version? It's OK if you did - just wanted to point out that v6 is out and if you have the time for a small challenge I'd use v6 so you're more in tune with the latest and greatest. 

that being said, it's totally optional! It operates differently than v5 so you'd have to invest a little bit of time reading docs on how to use it

Reply:
staying with v5

Reply:
great! In future scenarios, add a quick explanation, something like

"staying with v5 for now as we don't have the time in this sprint to make the change to v6" 

not a great example but I think you get what I mean (?)